ghost/admin/app/components/gh-post-settings-menu.js

```diff
@@ -604,4 +604,16 @@ export default class GhPostSettingsMenu extends Component {
     setSidebarWidthVariable(width) {
         document.documentElement.style.setProperty('--editor-sidebar-width', `${width}px`);
     }
+
+    get canViewPostHistory() {
+        let showPostHistory = this.feature.postHistory === true
+            && this.post.lexical
+            && this.post.emailOnly === false;
```

Comment:
Could be written concisely as 
```suggestion
        let showPostHistory = this.feature.postHistory
            && this.post.lexical
            && !this.post.emailOnly;
```

Reply:
@prabodhmeshram thanks for the feedback, my personal preference is explicitness seeming though I know the types of the variables 👍